app/services/prophet_ratings/prediction_evaluator.rb

```
@@ -254,5 +266,45 @@ def export_team_residuals_to_csv(path: 'tmp/team_residuals.csv')
+        labels[i] = bucket
+        counts << values.size
+      end
+
+      g = Gruff::Bar.new
+      g.title = 'Z-score Distribution of Margin Predictions'
+      g.labels = labels
+      g.data(:ZScores, counts)
+      g.write('tmp/z_score_histogram.png')
+
+      Rails.logger.info '📊 Saved Z-score histogram to tmp/z_score_histogram.png'
+    end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `plot_margin_z_scores_histogram` is too high. [<17, 40, 13> 45.37/22]

(Metrics/AbcSize)